build.gradle

```diff
@@ -657,6 +657,46 @@ project(':iceberg-delta-lake') {
   }
 }
 
+project(':iceberg-bigquery') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    api project(':iceberg-api')
+    implementation project(':iceberg-common')
+    implementation project(':iceberg-core')
+
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+
+    implementation platform(libs.google.libraries.bom)
+    compileOnly "com.google.cloud:google-cloud-storage"
+    implementation "com.google.cloud:google-cloud-bigquery"
+    implementation "com.google.cloud:google-cloud-core"
+
+    compileOnly libs.hadoop3.common
```

Comment:
Note that Hadoop dependency will prevent Trino from adopting BigQuery metastore catalog since we banned those libraries in https://github.com/trinodb/trino/issues/15921.

Reply:
Trino is important. After mergining this. I would like to give a support Trino too. I removed all hadoop and hive dependency except test. Is that OK @ebyhr ?